modules/widgets/facebook-likebox.php

```diff
@@ -42,6 +37,8 @@ function widget( $args, $instance ) {
 
 		$like_args = $this->normalize_facebook_args( $instance['like_args'] );
 
+		wp_enqueue_style( 'jetpack_facebook_likebox', plugins_url( 'facebook-likebox/style.css', __FILE__ ) );
```

Comment:
`maybe_inline_style` works for this. Added it https://github.com/Automattic/jetpack/commit/c191a229648b326f9cd7b882231c3097500869a5